governance/votes/Draft- Executive vote - September 13, 2023.md

```diff
@@ -0,0 +1,218 @@
+*Authorizing party*:  [Stability Scope Facilitator](http://forum.makerdao.com/t/request-to-poll-offboarding-legacy-legal-recourse-assets/21582/8)
+
+The Stability Scope Artifact requires that legacy Legal Recourse Assets be offboarded prior to the launch of NewChain. The Stability Scope Facilitator seeks approval of a DAO Resolution that will instruct the RWA Foundation to terminate future purchases in HV Bank’s RWA Master Participation Trust and return excess available cash to the DAO. 
+
+If this executive proposal passes, the DAO Resolution with the IPFS hash [QmXU2TwsRpVevGY74NVFbD9bKwtsw1mSuSce7My1zinD9m](https://gateway.pinata.cloud/ipfs/QmXU2TwsRpVevGY74NVFbD9bKwtsw1mSuSce7My1zinD9m) will be approved.
+
+For reference, a raw version of the text may be found below:
+
+```
+MAKERDAO RESOLUTION
+
+Date: DATED AS OF RELATED EXECUTIVE VOTE ON MAKERDAO
+No: 1
+Purpose: RWA MASTER PARTICIPATION TRUST TERMINATION OF FUTURE PURCHASES
+
+This resolution is intended to terminate future purchases into the RWA Master Participation Trust, established July 12, 2022 pursuant to the Amended and Restated Trust Agreement by and between Wilmington Savings Fund Society, FSB (in its capacity as “Trustee”) and RWA Foundation, a Cayman Islands foundation company (the “RWA Foundation).
+
+On July 12, 2022, RWA Master Participation Trust entered into a Master Participation Agreement (the “Agreement”) with Huntingdon Valley Bank, a Pennsylvania chartered bank, acting as “Seller” and RWA Master Participation Trust acting as “Buyer.”
+
+Reference is made to the Articles of Association (the “Articles”) of the RWA Foundation, pursuant to which RWA Foundation’s operations are governed.
+
+This resolution sets out the instructions and ratifications duly passed as an executed vote on https://vote.makerdao.com (“Maker Governance Voting Portal”) in accordance with the governance protocols of MakerDAO on the date specified above.
+
+RATIFICATION AND INSTRUCTION
+
+In accordance with the governance protocols of MakerDAO on the date specified above and pursuant to the Articles, the Maker token holders hereby approve and ratify the termination of future purchases into the RWA Master Participation Trust pursuant to the Agreement and instruct the directors of the RWA Foundation and its associated agents (Leeward Management, Zedra Directors, and Silverside Management) to perform the following actions:
+
+(i) Instruct the Trustee or other parties as necessary to provide written notice to Seller for the purpose of terminating future purchases as soon as permissible under the Agreement.
+(ii) Instruct the Trustee or other parties as necessary, after the termination of future purchases is in effect, to return excess available cash held at WSFS for the benefit of the RWA Master Participation Trust to the MakerDAO protocol
+
+```
+
```

Comment:
In general, it's good practice to eliminate excessive lines. While the information will appear the same on the portal, it's easier to review without unnecessary spacing.

Reply:
Thanks will remove!